test/src/596-catalogbalancer/main

```diff
@@ -0,0 +1,156 @@
+cvmfs_test_name="Catalog balancer"
+cvmfs_test_autofs_on_startup=false
+
+add_files() {
+  local repo_dir=$1
```

Comment:
It's the absolute path in the machine where the repository is actually located. It doesn't really make sense to be the root of the repository, because it's obviously the empty string. Anyway it could be renamed to... repo_abs_path? Suggestions are welcome :)